puppet/modules/slave/manifests/unittests.pp

```diff
@@ -168,4 +168,12 @@
 
   # RVM
   include slave::rvm
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '9' {
+    include slave::rvm
+  }
+
+  # rbenv
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '7' {
+    include slave::rbenv
```

Comment:
Intent is to add this to all our EL8 nodes, switch our code to it and then we can drop RVM and then update to EL9.